ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/application/validators/IPathValidator.java

```diff
@@ -0,0 +1,9 @@
+package edu.usb.argos.ASTProcessor.application.validators;
+
+import java.nio.file.Path;
+
+public interface IPathValidator {
```

Comment:
For consistency with others interfaces, IPathValidator could be IPathValidator<T> to handle different types of paths

Reply:
done